client/patches/frappe-gantt+0.6.1.patch

```diff
@@ -0,0 +1,2537 @@
+diff --git a/node_modules/frappe-gantt/src/bar.js b/node_modules/frappe-gantt/src/bar.js
```

Comment:
As an initial comment, we won't want to patch dependencies this way with such large changes -- I would recommend forking the project and pushing it to npm with the changes you need as a separate project, or even using a git dependency like:
```
  "dependencies": {
    "frappe-gantt": "github:yourgithub/frappe-gantt#123kj23kj",
```

Reply:
Cool, will work on it then, thanks for this input :)

Reply:
Git dependecy is now added to the feature :100: